src/runtime_safe_heap.js

```diff
@@ -14,7 +14,7 @@
     @param {number|boolean=} noSafe */
 function setValue(ptr, value, type, noSafe) {
   type = type || 'i8';
-  if (type.charAt(type.length-1) === '*') type = 'i32'; // pointers are 32-bit
+  if (type.charAt(type.length-1) === '*') type = `${Runtime.POINTER_TYPE}`;
```

Comment:
We really should just get rid of runtime and so we can just write `POINTER_TYPE` here.. but that is another change.

Reply:
I have reduced the need for it already, and now refer to these variables without the `Runtime.` (they come from `parseTools.js` instead.